### DISCUSSION TITLE
Adjusting Coordinates

### DISCUSSION THREAD

**Discussion by OakLoaf** [javalent/obsidian-leaflet#442] [Q&A] [answered]
I was wondering if it's possible to adjust the mapping of coordinates, my map is a minecraft map so it'd be nice if I could adjust where the coordinates are centred/the scale of the coordinates. 
Is this possible currently?

**Comment by valentine195** [comment]
You should be able to do that with the `bounds` parameter. 

https://plugins.javalent.com/leaflet/bounds

**Reply by OakLoaf** [reply]
Thank you, I'll give this a crack :)

**Reply by OakLoaf** [reply]
> You should be able to do that with the `bounds` parameter.
> 
> https://plugins.javalent.com/leaflet/bounds

I gave `bounds` a go but didn't have much luck, I tried setting it as below but when loading the map it appears that I can no longer see the image at all it just provides a blank map. 

I read in another section that lat and long should be percentages, does this apply here too? In that case what would I need to do to make the coordinates of the image as shown below?
```
bounds:
- -2048, -2048
- 2175, 2175
```

**Reply by valentine195** [reply]
Can you provide the map to me? I'll see if I can work it out.

**Reply by OakLoaf** [reply]
It's too large for github so I've posted it on imgur: https://imgur.com/Dcc52g6
The top left coordinate is `-2048, 2048` and bottom right is `2175, 2175`

Thank you very much for helping!

**Reply by OakLoaf** [reply]
Would it be possible to re-open this until we've resolved it?

**Reply by OakLoaf** [reply]
@valentine195 did you happen to get a chance to try this?

**Reply by valentine195** [reply]
Not yet, sorry. I will try to tomorrow

**Reply by valentine195** [reply, marked as answer]
@OakLoaf Hey I just took a closer look at this; it's not rendering because the bounds need to be an array. Try these bounds:

```yaml
bounds:
  - [-2048, -2048]
  - [2175, 2175]
```

**Reply by OakLoaf** [reply]
Apologies for the late reply to this, that worked, only issue I have now is that I can't zoom very far out when looking at the map, is it possible to adjust that?

**Reply by OakLoaf** [reply]
Hey @valentine195, should I open another issue regarding this one or is it ok in here?
